PostFreeSurfer/PostFreesurferPipelineLongPrep.sh

```diff
@@ -0,0 +1,434 @@
+#!/bin/bash
+
+# # PreFreeSurferPipeline.sh
+#
+# ## Copyright Notice
+#
+# Copyright (C) 2022-2024 The Human Connectome Project
+#
+# * Washington University in St. Louis
+# * University of Minnesota
+# * Oxford University
+#
+# ## Author(s)
+#
+# * Mikhail Milchenko, Department of Radiology, Washington University in St. Louis
+# * Matthew F. Glasser, Department of Anatomy and Neurobiology, Washington University in St. Louis
+#
+# ## Product
+#
+# [Human Connectome Project][HCP] (HCP) Longitudinal Pipelines
+#
+# ## License
+#
+# See the [LICENSE](https://github.com/Washington-University/Pipelines/blob/master/LICENSE.md) file
+
+
+function create_zero_warp
```

Comment:
Please put functions and anything else after the argument parsing.  We want the argument stuff to be close to the top of the script.

Reply:
ok